testing/sdk_cmd.py

```diff
@@ -27,6 +29,6 @@ def run_cli(cmd, print_output=True):
     if ret != 0:
         err = 'Got error code {} when running command "dcos {}":\nstdout: "{}"\nstderr: "{}"'.format(
             ret, cmd, stdout, stderr)
-        sdk_utils.out(err)
+        log.info(err)
```

Comment:
Instead of using `info()` here, one could use `error`. This does not terminate, but does provide some extra context.